apps/frontend/lib/clients/cloudflare-r2.ts

```diff
@@ -145,4 +147,14 @@ async function putObject(
   return completeResponse;
 }
 
-export { getObjectByQuery, putObjectByQuery };
+async function copyObject(source: ObjectLocation, destination: ObjectLocation) {
+  const command = new CopyObjectCommand({
+    CopySource: encodeURIComponent (`/${source.bucketName}/${source.objectKey}`),
```

Comment:
```suggestion
    CopySource: encodeURIComponent(`${source.bucketName}/${source.objectKey}`),
```

I don't think there's a `/` prefix

Reply:
@IcaroG